src/components/language/language.service.ts

```diff
@@ -450,7 +450,7 @@ export class LanguageService {
         response[record.property] = record.value;
       } else {
         response[record.property] = {
-          value: record.value || null,
+          value: record.value ?? null,
```

Comment:
This returns the value if it exists, but we need to only return the value if the user has canRead=true.

Reply:
`false || null` is `null` so it will raise the https://github.com/SeedCompany/cord-api-v3/issues/978 issue.
So if we only want to return value if the `canRead` is `true` then:
```
const canRead = perms[record.property] ? perms[record.property].canRead : false;
response[record.property] = {
   value: canRead ? record.value : null,
   canRead: canRead,
   ...
};
```
How do you think about this change? @michaelmarshall